AgentDiscoveries-Frontend/app/src/components/message.js

```diff
@@ -0,0 +1,12 @@
+import * as React from "react";
+
+export class Message extends React.Component {
+
+    render() {
+        return (
+            <div className={`${this.props.message.type}-message`}>

```

Comment:
React as some build in alert stuff which might make this a bit neater? https://react-bootstrap.github.io/components/alerts/